### PR TITLE
Mrna clean clustering

### LINKED ISSUE #29: Global Clustering
Clean up code of binary dendrogram as a figure quality pdf

### PR DESCRIPTION
**Pull request by soraya-s**

### CODE REVIEW

File: analysis/02_global_clustering/global_clustering.Rmd [hunk 1]
```diff
@@ -86,7 +83,7 @@ hist(promoter_positive_peaks)
```
Comment (msmallegan) [inline analysis/02_global_clustering/global_clustering.Rmd:70]: Looks there's still a bit to clean up here, for ex. line 79 still has HINT. We'll wait to merge it until all this is cleaned up.

**Review by msmallegan — CHANGES_REQUESTED** [review]

**Review by shle4568 — APPROVED** [review]
I made additional deletions to clean the code, am now confused on how to do a new pull request with the resulting code/how to resolve this from within the existing pull request. Hopefully this fixes things and doesn't mess anything up!

File: analysis/00_class_exercises/mrna_group/enriched_factors_on_reservoirs.Rmd [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+total <- total_ghosts + total_not_ghosts
+
+chisq_res <- data.frame("tf" =character(),
+                        "ghosted" = numeric(),
+                        "chisq_stat" = numeric(),
+                        "chisq_pval" = numeric(),
+                        "not_ghosts" = numeric())
+
+i <- 1
+## How do we find expected number of ghosts??
```
Comment (boulderrinnlab) [inline analysis/00_class_exercises/mrna_group/enriched_factors_on_reservoirs.Rmd:130]: I would take one more pass on markdown and clean up

File: analysis/00_class_exercises/mrna_group/enriched_factors_on_reservoirs.Rmd [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+peak_occurence_df[which(peak_occurence_df$number_of_tfs > x), "tf_binding"] <- "high"
+
+g <- ggplot(peak_occurence_df, aes(x = log10(tpm), color = tf_binding))
+g + geom_density(alpha = 0.2)
+```
+
+
+```{r}
+# Here's an example of a binned approach.
+## Are these reasonable break points? How would consider changing them?
```
Comment (boulderrinnlab) [inline analysis/00_class_exercises/mrna_group/enriched_factors_on_reservoirs.Rmd:196]: Clean up markdown

**Pull request closed without merging**